docs/components/movement-sensor/gps/gps-nmea-rtk-serial.md

```diff
@@ -102,15 +101,15 @@ Edit and fill in the attributes as applicable.
+| `serial_baud_rate`       | int    | Optional     | The rate at which data is sent from the sensor. <br> Default: `38400`                                                                                                                                                                                                                                                                                                                                    |
+| `ntrip_url`              | string | **Required** | The URL of the NTRIP server from which you get correction data. Connects to a base station (maintained by a third party) for RTK corrections.                                                                                                                                                                                                                                                            |
+| `ntrip_username`         | string | Optional     | Username for the NTRIP server.                                                                                                                                                                                                                                                                                                                                                                           |
+| `ntrip_password`         | string | Optional     | Password for the NTRIP server.                                                                                                                                                                                                                                                                                                                                                                           |
+| `ntrip_connect_attempts` | int    | Optional     | How many times to attempt connection before timing out. <br> Default: `10`                                                                                                                                                                                                                                                                                                                               |
+| `ntrip_mountpoint`       | string | Optional     | If you know of an RTK mountpoint near you, write its identifier here. It will be appended to NTRIP address string (for example, "nysnet.gov/rtcm/**NJMTPT1**") and that mountpoint's data will be used for corrections.                                                                                                                                                                                  |
```

Comment:
```suggestion
| ------------------------ | ------ | ------------ | ---------------- |
| `serial_path`            | string | **Required** | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyS0`, to its device file, such as <file>/dev/ttyS0</file>. If you omit this attribute, Viam will attempt to automatically detect the path. |
| `serial_baud_rate`       | int    | Optional     | The rate at which data is sent from the sensor. <br> Default: `38400` |
| `ntrip_url`              | string | **Required** | The URL of the NTRIP server from which you get correction data. Connects to a base station (maintained by a third party) for RTK corrections. |
| `ntrip_username`         | string | Optional     | Username for the NTRIP server. |
| `ntrip_password`         | string | Optional     | Password for the NTRIP server. |
| `ntrip_connect_attempts` | int    | Optional     | How many times to attempt connection before timing out. <br> Default: `10` |
| `ntrip_mountpoint`       | string | Optional     | If you know of an RTK mountpoint near you, write its identifier here. It will be appended to NTRIP address string (for example, "nysnet.gov/rtcm/**NJMTPT1**") and that mountpoint's data will be used for corrections. |
```